recipes/hkl/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "hkl" %}
+{% set version = "5.0.0.2173" %}
+{% set sha256 = "5f3991d2af4ff01d6f6e29d0176af9bc1b54e6a8598046a050ad804fedd860f2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/picca/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py<36 or win or osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - autoconf
+    - automake
+    - libtool
+    - m4
+    - pkg-config
+  host:
+    - glib
+    - gobject-introspection
+    - gsl
+    - pygobject
+    - python
+  run:
+    - python
+    - gsl
+    - libiconv
+    - numpy
+    - pcre
+    - pygobject
+
+test:
+  commands:
+    - test -f ${PREFIX}/lib/libhkl.a
```

Comment:
Please do not package the static library

Reply:
This was addressed.